assets/src/scripts/web-services/groundwater-levels.js

```diff
@@ -7,18 +7,25 @@ import config from 'ui/config';
  *  Formats a URL for the purpose of downloading groundwater levels data in RDB format
  * @param  {String}  siteno - A code that uniquely identifies a monitoring location
  * @param  {String}  parameterCode - USGS five digit parameter code
- * @param {String} startTimeDT - ISO 8601 time
```

Comment:
This is actually what caused the issue with the display of groundwater levels with custom time. Also standardizing the variable name as startTime/endTime as these are times (not just dates). We shouldn't let the misleading naming on the API leak into the code.

Reply:
Ah, good catch. I'm still trying to learn all the time/date formatting options.